zqd/space/archivespace.go

```diff
@@ -0,0 +1,162 @@
+package space
```

Comment:
Nit: I think archive.go would be a better file name since this file defines both archiveSpace and archiveSubspace.